modules/apps/archived/archived-modules-upgrade/src/main/resources/content/Language.properties

```diff
@@ -1,7 +1,9 @@
 deprecated-modules-upgrade-configuration-name=Deprecated Modules Upgrade
 remove-chat-module-data=Remove chat module data during upgrade.
+remove-dictionary-module-data=Remove dictionary module data during upgrade.
```

Comment:
@SamZiemer we need to execute "gw buildLang" to generate the translations for this keys. See this if you get any error:
https://liferay.slack.com/archives/C04D8HMEF/p1588276467112500